definitions/ext-access_point/definition.yml

```diff
@@ -1,22 +1,37 @@
 domain: EXT
 type: ACCESS_POINT
 synthesis:
-  name: device_name
-  identifier: device_name
-  encodeIdentifierInGUID: true
-
-  conditions:
-  - attribute: provider
-    value: kentik-wap
-
-  tags:
-    src_addr:
-      entityTagName: device_ip
-      multiValue: false
-
+  rules:
+    # kentik-provider
+    - identifier: device_name
+      name: device_name
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: provider
+          value: kentik-wap
+      tags:
+        src_addr:
+          entityTagName: device_ip
+          multiValue: false
+   # prometheus provider         
+    - identifier: serial
+      name: ip
+      debug: true 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: MetricRaw
+        - attribute: metricName
+          value: 'unifipoller_device_info'
+      tags:
+        model:
+        version:
+        mac:
+        ip:
+          entityTagName: device_ip
+          multiValue: false  
 goldenTags:
 - device_ip
-
 dashboardTemplates:
```

Comment:
You likely want a dashboard for this provider right?

Reply:
Not yet. This is only to debug the test rules